specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyEvents.json

```diff
@@ -0,0 +1,1024 @@
+        "resourceTags": {
+          "description": "List of resource tags.",
+          "type": "string"
+        },
+        "policyAssignmentName": {
+          "description": "Policy assignment name.",
+          "type": "string"
+        },
+        "policyAssignmentOwner": {
+          "description": "Policy assignment owner.",
+          "type": "string"
+        },
+        "policyAssignmentParameters": {
+          "description": "Policy assignment parameters.",
+          "type": "string"
+        },
+        "policyAssignmentScope": {
+          "description": "Policy assignment scope.",
+          "type": "string"
+        },
+        "policyDefinitionName": {
+          "description": "Policy definition name.",
+          "type": "string"
+        },
+        "policyDefinitionAction": {
+          "description": "Policy definition action, i.e. effect.",
+          "type": "string"
+        },
+        "policyDefinitionCategory": {
+          "description": "Policy definition category.",
+          "type": "string"
+        },
+        "policySetDefinitionId": {
+          "description": "Policy set definition ID, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "policySetDefinitionName": {
+          "description": "Policy set definition name, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "policySetDefinitionOwner": {
+          "description": "Policy set definition owner, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "policySetDefinitionCategory": {
+          "description": "Policy set definition category, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "policySetDefinitionParameters": {
+          "description": "Policy set definition parameters, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "managementGroupIds": {
+          "description": "Comma separated list of management group IDs, which represent the hierarchy of the management groups the resource is under.",
+          "type": "string"
+        },
+        "policyDefinitionReferenceId": {
+          "description": "Reference ID for the policy definition inside the policy set, if the policy assignment is for a policy set.",
+          "type": "string"
+        },
+        "complianceState": {
+          "description": "Compliance state of the resource.",
+          "type": "string"
+        },
+        "tenantId": {
+          "description": "Tenant ID for the policy event record.",
+          "type": "string"
+        },
+        "principalOid": {
+          "description": "Principal object ID for the user who initiated the resource operation that triggered the policy event.",
+          "type": "string"
+        },
+        "components": {
+          "description": "Components event records. By default this is not populated.",
```

Comment:
>By default this is not populated. [](start = 52, length = 33)

Same comment as policyStates.  Please specify when this is populated

Reply:
Sure, fixing it now.